docs/user-manual.md

```diff
@@ -0,0 +1,100 @@
+
+# Cambridge Particle Tracks - User's manual
+
+<!-- TOC start -->
+
+- [Introduction](#introduction)
+- [Usage](#usage)
+    * [Starting the tool](#start)
+    * [Loading the data](#load)
+    * [Adding a new particle](#newparticle)
+    * [Measuring particle properties](#measure)
+        + [Decay length](#length)
+        + [Radius of curvature](#radius)
+        + [Decay angles](#angles)
+        + [Stereoshift](#stereoshift)
+    * [Measuring the image magnification](#magnification)
+    * [Saving the data](#save)
+- [Useful keyboard shortcuts](#tips)
+- [References](#references)
+
+
+<!-- TOC end -->
+
+<!-- TOC --><a name="introduction"></a>
```

Comment:
```suggestion
```

Because:
```
 Warning, treated as error:
/home/runner/work/cavendish-particle-tracks/cavendish-particle-tracks/docs/user-manual.md:6:'myst' cross-reference target not found: 'introduction' [myst.xref_missing]
```
I've never seen a TOC like this. Was it automatically converted from LaTeX or something? 

I'm pretty sure our template automatically puts a TOC to the right on the webpage(?) If not, we could change templates...

Propose to just use `# Title` / `## Subtitle` / `### Sub sub title` rules and we can figure out the TOC later.

Reply:
I did not know how to get the TOC, so I googled this ugly thing... I'll remove it for the moment. Thanks!